solr/solrj/src/test/org/apache/solr/client/solrj/io/TupleTest.java

```diff
@@ -0,0 +1,101 @@
+package org.apache.solr.client.solrj.io;
```

Comment:
minor: the pull request CI or locally `gradlew precommit` or `gradlew checks` will likely flag that there's no license header here as yet